modules/web-console/e2e/testcafe/fixtures/admin/assume-identity.js

```diff
@@ -14,30 +14,31 @@
  * limitations under the License.
  */
 
-import { dropTestDB, insertTestUser, resolveUrl } from '../../environment/envtools';
-import { createRegularUser } from '../../roles';
+import { dropTestDB, insertFullTestUser, resolveUrl } from '../../environment/envtools';
+import { prepareUser, cleanupUser } from '../../roles';
 import * as admin from '../../page-models/pageAdminListOfRegisteredUsers';
 import {pageProfile as profile} from '../../page-models/pageProfile';
 import * as notifications from '../../components/permanentNotifications';
 import {userMenu} from '../../components/userMenu';
 
-const regularUser = createRegularUser();
-
 fixture('Assumed identity')
-    .beforeEach(async(t) => {
-        await dropTestDB();
+    .before(async() => {
         await dropTestDB(admin.TEST_USER.email);
-        await insertTestUser();
-        await insertTestUser(admin.TEST_USER);
-        await t.useRole(regularUser);
+        await insertFullTestUser(admin.TEST_USER);
+    })
+    .beforeEach(async(t) => {
+        await prepareUser(t);
+    })
+    .afterEach(async(t) => {
+        await cleanupUser(t);
     })
-    .afterEach(async() => {
+    .after(async(t) => {
         await dropTestDB(admin.TEST_USER.email);
```

Comment:
Why do you have to cleanup everywhere? Shouldn't a single cleanup in `afterEach` or `after` suffice?